Automattic/woocommerce-payments
# Move PaymentInformation to WCPay namespace

Fixes #864 

#### Changes proposed in this Pull Request

* Move `PaymentInformation` out of the `WCPay\DataTypes` namespace and into the `WCPay` namespace.
* Move `class-payment-information.php` out of `includes/data-types` and into `includes/`.
* Delete the (now obsolete) `data-types` folder.

#### Testing instructions

* Make sure unit tests pass: `npm run test:php`.
* Make sure you can start a new subscription.
* Make sure you can renew a subscription as a customer.
* Make sure you can process a renewal as an admin.

-------------------
